.gitignore

```diff
@@ -130,3 +130,7 @@ dmypy.json
 
 # Typings generated by VSCode
 typings/**
+
+# Libraries
+!libbls.so
+!libbls.dylib
```

Comment:
this one is not committed. expected?

Reply:
Also, I am missing the entry in `.gitignore` thay actually would ignore it.
Maybe add each exception under the intruction that actually ignores it. For example:
```
*.so
!libbls.so
```

Reply:
The ignore rule is a bit above, in the "standard" rules:

```
# C extensions
*.so
```

I've now committed the MacOS library, as well (was unavailable at that time).

Both libraries were taken from here: https://github.com/ElrondNetwork/elrond-sdk-erdgo/releases/tag/v1.2.0

```
$ sha256sum ./libbls.so 
e96982164e92a5dd616256d3a3d693c5f62934039f47418c6163cd5b72bd4ba4  ./libbls.so
$ sha256sum ./libbls.dylib 
2ad6344aa3b185ee400dc02ff582b730b65e538461c025c64124ff7ed22fb3ed  ./libbls.dylib
```

Reply:
Added files (for Linux, MacOS and Windows).